enterprise-suite/scripts/lbc.py

```diff
@@ -214,11 +215,40 @@ def check_credentials(creds):
         else:
             printerr('error: check_credentials failed: {}'.format(err))
     finally:
-        return success 
+        return success
 
-def preinstall_check(creds, minikube=False, minishift=False):
+
+
+url_http_timeout=3 # 3 seconds
```

Comment:
Probably good idea to have http request timeout, but then it should also be applied in check_credentials().